libraries/Microsoft.Bot.Connector/Authentication/ICredentialProvider.cs

```diff
@@ -54,5 +55,7 @@ public interface ICredentialProvider
         /// that may need to call out to serviced to validate the appId / password pair.
         /// </para></remarks>
         Task<bool> IsAuthenticationDisabledAsync();
+
+        ServiceClientCredentials GetCredentials();
```

Comment:
this is a breaking change, we have to do a different way

Reply:
This has been moved to ICredentialProvider2